CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ Changelog
 
 ### Bugfixes
 
+- Updated `examples/meyer_penny_game.py` with the correct path to the Meyer
+  Penny game exercise in `docs/source/exercises.rst` (gh-1045, @appleby).
```

Comment:
```suggestion
  Penny game exercise in `docs/source/exercises.rst` (@appleby, gh-1045).
```

Reply:
Whoops, fixed. I also switched to 100 cols wrapping.